Resources/Storage/GFAL2_StorageBase.py

```diff
@@ -319,31 +334,31 @@ def __putSingleFile( self, src_file, dest_url, sourceSize ):
       # no checksum check, compare file sizes for verfication
       else:
         res = self.__getSingleFileSize( dest_url )
-        if res['OK']:
-          destSize = res['Value']
-        else:
-          destSize = None
-        self.log.debug( 'GFAL2_StorageBase.__putSingleFile: destSize: %s, sourceSize: %s' % ( destSize, sourceSize ) )
+        if not res['OK']:
+          return res
```

Comment:
If there is a return here the cleanup of the failed transfer will not happen, or will this always happen when an error is returned from this function?